src/components/interfaces/HMI_API.xml

```diff
@@ -503,6 +503,45 @@
     <element name="turnText"/>
 </enum>
 
+<enum name="TextFieldType">
```

Comment:
Changes to the proposal regarding the naming of these structs/enums were agreed upon in the last SDLC meeting: https://github.com/smartdevicelink/sdl_evolution/pull/239
Change to `MetadataType`